radioactivedecay/decayenergy.py

```diff
@@ -0,0 +1,109 @@
+"""
+The decayenergy module loads in the decay energies data sets, and defines two key functions for
+calculating decay energy
+
+All data comes from this website: https://www-nds.iaea.org/relnsd/vcharthtml/VChartHTML.html#dcy2
+
+The docstring code examples assume that ``radioactivedecay`` has been imported
+as ``rd``:
+
+.. highlight:: python
+.. code-block:: python
+
+    >>> import radioactivedecay as rd
+
+Constants
+---------
+EV_TO_WH : conversion from electron volts to watt hours
+DECAY_ENERGIES: Map of isotopes to the energy the release per decay type
+META_ENERGIES: Map of isotopes metastable version to the energy they release 
+    when returning to the normal state
+MODE_MAP: map that converts the decay modes found in this package 
+    to the types used by the source data
+"""
+
+from importlib import resources
+import math
+import pickle
+from typing import Dict, List, Tuple
+
+
+EV_TO_WH = 1.602176634 * 10e-19
```

Comment:
do you mean to offer a Wh value or Joules value? If Wh I think you need to divide by 3600 here

Reply:
Related to this line, scipy has a submodule of physical [constants](https://docs.scipy.org/doc/scipy/reference/constants.html#). Although likely that most people looking at the file will recognise this number, why be explicit and use the name? As scipy is already import in another location we would not be importing it just for access to this value.